cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -1234,7 +1234,16 @@ void SbPlayerPipeline::OnPlayerError(SbPlayerError error,
       CallErrorCB(::media::PIPELINE_ERROR_DECODE, message);
       break;
     case kSbPlayerErrorCapabilityChanged:
+#if SB_API_VERSION < 16
       CallErrorCB(::media::PLAYBACK_CAPABILITY_CHANGED, message);
+#else
+      CallErrorCB(::media::PIPELINE_ERROR_DECODE,
```

Comment:
nit: it's good to add a comment here to explain that we're removing non standard capability changed error.

And also let's notify h5 player team that we'll change capability changed error as they may need to refine their handling.

Reply:
We should probably ask the h5 player team to add the handling of the new error message now, in that case we may be able to remove all the `#if SB_API_VERSION < 16` guard in this PR.

Reply:
The H5 PR is in, so I've removed all the guards.